websocketOutportDriver/tests/realtest/server/server.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go-core/core/mock"
+	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
+	"github.com/ElrondNetwork/elrond-go-core/data/typeConverters/uint64ByteSlice"
+	"github.com/ElrondNetwork/elrond-go-core/marshal"
+	"github.com/ElrondNetwork/elrond-go-core/websocketOutportDriver/data"
+	"github.com/ElrondNetwork/elrond-go-core/websocketOutportDriver/factory"
+)
+
+var jsonMarshaller = &marshal.JsonMarshalizer{}
+
+func main() {
+	server, err := createServer()
+	if err != nil {
+		fmt.Println("cannot create server: ", err.Error())
+		return
+	}
+
+	timeoutChan := make(chan bool)
+	go func(tChan chan bool) {
+		time.Sleep(5 * time.Minute)
+		tChan <- true
+	}(timeoutChan)
+
+	for {
+		select {
+		case <-timeoutChan:
+			return
+		default:
```

Comment:
instead of default you could have `case <-time.After(time.Second)` and removed L35